src/app/workflow/descriptors/descriptors.component.ts

```diff
@@ -35,6 +35,7 @@ import { FilesService } from '../files/state/files.service';
 export class DescriptorsWorkflowComponent extends EntryFileSelector {
   @Input() id: number;
   @Input() entrypath: string;
+  @Input() otherFiles: boolean;
```

Comment:
so descriptors is going to show OTHER files then?

Reply:
If true, shows only other files, if false, shows descriptor files. I just didn't want to clone an entire component just for the other files.